waku/v2/protocol/waku_lightpush/waku_lightpush_types.nim

```diff
@@ -0,0 +1,32 @@
+import
+  std/[tables],
+  bearssl,
+  libp2p/peerinfo,
+  libp2p/protocols/protocol,
+  ../../node/peer_manager/peer_manager,
+  ../waku_message
+
+export waku_message
+
+type
+  PushRequest* = object
+    pubsubTopic*: string
```

Comment:
Ah. Minor note, but had a little debate with myself earlier re `pubsubTopic` vs `pubSubTopic` and couldn't find a consensus elsewhere. My vote is `pubSubTopic`, unless we've made a decision on another thread? cc @staheri14

Reply:
sure thing, fixed! (didn't touch wakunode2 and bridge to keep it self-contained)